include/ublk_cmd.h

```diff
@@ -359,6 +359,10 @@ struct ublksrv_ctrl_dev_info {
  * include/uapi/linux/blkzoned.h are part of ublk UAPI.
  */
 #define		UBLK_IO_OP_REPORT_ZONES		18
+#define		UBLK_IO_OP_EPOLLFD		20
```

Comment:
UBLK_IO_OP_EPOLLFD needs to move to include/ublksrv.h, and define in the reserved range